ironfish-cli/package.json

```diff
@@ -43,7 +43,7 @@
     "lint": "tsc -b && eslint --ext .ts,.tsx,.js,.jsx src/",
     "lint:fix": "tsc -b && eslint --ext .ts,.tsx,.js,.jsx src/ --fix",
     "start:dev": "node start",
-    "start": "yarn build && yarn start:js",
+    "start": "yarn --cwd ../ build && yarn start:js",
```

Comment:
Man this is really nice for fixing end user problems, but it does make running it a lot slower. I'll have to edit this out when developing myself. I think a lot of nodes-guru people who run this for all their commands will also get a lot slower.

Reply:
That is a trade-off, for sure. I don't think I re-run commands often enough that the slowdown will have much affect on me, but I suppose everyone's workflow is different. I'm fine not merging this if we prefer.